### ISSUE TITLE
double_space: should not error for inline declarations

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#2206]
```abap
REPORT zfoobar.

DATA your_variable TYPE i.
DATA your_var_01 TYPE i.
DATA your_var_2 TYPE i.

your_variable = 0.
your_var_01   = 0. " ok
your_var_2    = 0. " ok

DATA(my_variable) = 0.
DATA(my_var_01)   = 0. " remove double space
DATA(my_var_2)    = 0. " remove double space
```

It should not raise error in this case.

**Comment by larshp** [comment]
@mbtools try again, latest version

**Comment by mbtools** [comment]
👍
